.github/workflows/deploy_rovecomm.yml

```diff
@@ -35,13 +38,35 @@ jobs:
           echo $tag
           git checkout $tag
 
+      - name: Build and Create Executable
+        run: |
+          cd /opt/RoveComm_CPP/
+          if [ -d "build" ]; then rm -Rf build; fi
+          mkdir build
+          cd build
+          cmake ..
+          make
+          make package
+
+      - name: Rename Executable
+        run: mv /opt/RoveComm_CPP/build/RoveComm_CPP**.sh /opt/RoveComm_CPP/build/RoveComm_CPP_AMD64.sh
+
+      - name: Upload Artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: RoveComm-AMD64
+          path: /opt/RoveComm_CPP/build/RoveComm_CPP_AMD64.sh
+
   create-release:
     runs-on: ubuntu-latest
     needs: build-and-upload
 
     steps:
       - name: Checkout repository
```

Comment:
Step not needed

Reply:
Since it's just building the AMD64 package and it's not using the self hosted runners, /opt/RoveComm_CPP doesn't exist in the runner be default. I previously created a release tag and the action failed to run saying it couldn't cd into the repo path. I figured it wasn't cloning the repo into the correct location, but I'm not super sure.

Latest release action run is [here](https://github.com/MissouriMRDT/RoveComm_CPP/actions/runs/8240694311/job/22536560995)

Reply:
The action failed up in the **build-and-upload** job on line 43 of the file. I am having you remove a step in the **create-release** job since all you are doing is copying the upload artifacts.